io/subsystem/src/main/java/org/wildfly/extension/io/WorkerResourceDefinition.java

```diff
@@ -74,6 +74,12 @@
     static final RuntimeCapability<Void> IO_WORKER_RUNTIME_CAPABILITY =
             RuntimeCapability.Builder.of(IOServices.IO_WORKER_CAPABILITY_NAME, true, XnioWorker.class).build();
 
+
+    static final OptionAttributeDefinition WORKER_TASK_CORE_THREADS = new OptionAttributeDefinition.Builder(Constants.WORKER_TASK_CORE_THREADS, Options.WORKER_TASK_CORE_THREADS)
+            .setAllowExpression(true)
+            .setDefaultValue(new ModelNode(2))
```

Comment:
Can you please share some reasoning why is default value set to 2?

Reply:
we always had (this attribute was already present in 1.0 version of subsystem) that as 2, as a starting thread pool size sounds reasonable. 
We also discussed that with David and agreed that is a reasonable default.